source/_integrations/home_connect.markdown

```diff
@@ -32,6 +34,7 @@ The integration will add one Home Assistant device for each connected home appli
 
 - A power switch
 - If the device has programs, switches for each of the individual programs will be added. Note that program options cannot be configured currently.
+- If the device has programs, a select entity that will allow you to select the program between the available ones.
```

Comment:
We should mention and clarify the two different select entities for each program.